ci: Code coverage is broken

https://github.com/riboseinc/enprot/runs/650312557

```
+ sudo make install
python src/scripts/install.py --prefix="/usr" --build-dir="build" --bindir=/usr/bin --libdir=/usr/lib --docdir=share/doc --includedir=include
   INFO: Botan 2.13.0 installation complete
error: failed to run `rustc` to learn about target-specific information

Caused by:
  process didn't exit successfully: `rustc - --crate-name ___ --print=file-names -Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Zno-landing-pads --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro --print=sysroot --print=cfg` (exit code: 1)
--- stderr
error: unknown debugging option: `no-landing-pads`


##[error]Process completed with exit code 101.
```

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -61,7 +61,7 @@ jobs:
     env:
       CODECOV_TOKEN: 79f7256d-b5a9-4843-923f-43dceef2fdc1
       CARGO_INCREMENTAL: 0
-      RUSTFLAGS: -Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Zno-landing-pads
+      RUSTFLAGS: -Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests
     steps:
       - uses: actions/checkout@v1
       - name: Install rust nightly

